azure-pipelines.yml

```diff
@@ -311,3 +311,21 @@ jobs:
   - bash: ci/run_win.sh
     displayName: 'Testing'
 
+- job: PyPi
+  dependsOn:
+    - 'Ubuntu16Py37'
+    - 'Ubuntu16All'
+    - 'Ubuntu16Py27'
+    - 'Ubuntu16PyPy2'
+    - 'Ubuntu16PyPy3'
+    - 'OSX10_13'
+    - 'Win2016'
+  condition: and(succeeded(), eq(variables['build.sourceBranch'], 'refs/heads/master'))
+  steps:
+  - script: 'pip install twine wheel'
```

Comment:
Does the python version on which `pip` is executed matter? In case you need to use `UsePythonVersion@0` task, otherwise ignore this comment.